### ISSUE TITLE
mrbayes installation illegal instruction (core dumped)

### ISSUE THREAD

**Issue by kmtayaban** [NBISweden/MrBayes#150]
Last 15 lines from /home/kmtayaban/.cache/Homebrew/Logs/mrbayes/02.make:
2020-01-31 17:39:53 +0800

make

/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/make: line 25: exec: : not found

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/brewsci/homebrew-bio/issues

**Comment by andkaha** [comment]
To compile MrBayes using Linuxbrew, please see the section titled _Installing MrBayes on Linux or WSL using Homebrew ("Linuxbrew")_ in the `INSTALL` document:  https://github.com/NBISweden/MrBayes/blob/develop/INSTALL

If you have further issues, please respond to this issue and then also mention what Linux operating system you are working with.  Also make sure that your Linuxbrew installation is properly up to date and otherwise in working order.

**Comment by kmtayaban** [comment]
Hi,
The first time I installed and opened mrbayes using WSL, I got an "illegal instruction (core dumped)". So I followed the documentation and stil got this issue:
brew reinstall -s mrbayes
==> Reinstalling brewsci/bio/mrbayes
==> Downloading https://github.com/NBISweden/MrBayes/archive/v3.2.7a.tar.gz
Already downloaded: /home/kmtayaban/.cache/Homebrew/downloads/8c17446ecefe113fd63990d68282ee7efb54ab8f7c87e5596439300c6c14b62a--MrBayes-3.2.7a.tar.gz
==> ./configure --prefix=/home/linuxbrew/.linuxbrew/Cellar/mrbayes/3.2.7 --with-mpi=no
==> make
Last 15 lines from /home/kmtayaban/.cache/Homebrew/Logs/mrbayes/02.make:
2020-02-14 15:07:13 +0800

make

/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/make: line 25: exec: : not found

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/brewsci/homebrew-bio/issues

**Comment by andkaha** [comment]
This is not a MrBayes issue. It is most likely due to not having the `build-essential` package installed. This Linux package contains build tools such as `make` and the C compiler.

I just installed Linuxbrew from scratch on a Windows 10 laptop to make sure, and when doing so it informed me that I needed to do `sudo apt-get install build-essential` (I'm using Ubuntu on WSL, there are equivalent packages if you're using some other Linux distribution that is not using the `apt` package system).

I then had no issues compiling, installing, and running MrBayes from Linuxbrew.
